mne_connectivity/spectral/time.py

```diff
@@ -302,12 +391,42 @@ def spectral_connectivity_time(data, freqs, method='coh', average=False,
 
     # get indices of pairs of (group) regions
     if indices is None:
-        indices_use = np.tril_indices(n_signals, k=-1)
+        if multivariate_con:
+            if any(this_method in _gc_methods for this_method in method):
+                raise ValueError(
+                    'indices must be specified when computing Granger '
+                    'causality, as all-to-all connectivity is not supported')
+            logger.info('using all indices for multivariate connectivity')
+            indices_use = (np.arange(n_signals, dtype=int),
+                           np.arange(n_signals, dtype=int))
+        else:
+            logger.info('only using indices for lower-triangular matrix')
+            indices_use = np.tril_indices(n_signals, k=-1)
     else:
+        if multivariate_con:
+            if len(np.unique(indices[0])) != len(np.unique(indices[1])):
+                raise ValueError(
```

Comment:
Can you add a unit-test for this case?

Reply:
Done! https://github.com/tsbinns/mne-connectivity/blob/894c42e59cef9f97aa07a7a4f71cef756512c50c/mne_connectivity/spectral/tests/test_spectral.py#L1103-L1108